src/agent/messages.rs

```diff
@@ -12,12 +13,14 @@ pub enum Command {
     Initialize(usize, Vec<u32>),
     SetExecuteSpeed(u32),
     SetRegister(String, u64),
+    SetFPRegister(String, u64),
     SetMemory(u64, u32),
     Execute,
     ExecuteInstruction,
     ExecuteStage,
     Pause,
     Reset,
+    SetBreakpoint(u64),
```

Comment:
We can probably go over this in more detail in the meeting but the way I've been implementing breakpoints is through the BREAK instruction, meaning the emulator core is responsible for signalling when breakpoints were hit as opposed to the agent having to monitor the program counter of the emulator core and breaking when necessary. I think UI-wise this is better but implementing the breakpoints like this (as CPU instructions) is just way easier than setting breakpoints from the UI, so if we go down this route I'd rather do it at a later date when we're not in as much of a rush.